crates/types/src/light_client.rs

```diff
@@ -109,3 +119,100 @@ impl From<schnorr::KeyPair<Config>> for StateKeyPair {
         StateKeyPair(value)
     }
 }
+
+/// Request body to send to the state relay server
+#[derive(Clone, Debug, CanonicalSerialize, CanonicalDeserialize, Serialize, Deserialize)]
+pub struct StateSignatureRequestBody {
+    /// The public key associated with this request
+    pub key: StateVerKey,
+    /// The associated light client state
+    pub state: LightClientState,
+    /// The associated signature of the light client state
+    pub signature: StateSignature,
+}
+
+/// The state signatures bundle is a light client state and its signatures collected
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct StateSignaturesBundle {
+    /// The state for this signatures bundle
+    pub state: LightClientState,
+    /// The collected signatures
+    pub signatures: HashMap<StateVerKey, StateSignature>,
+    /// Total stakes associated with the signer
+    pub accumulated_weight: U256,
+}
+
+/// Public input to the light client state prover service
+#[derive(Clone, Debug)]
+pub struct GenericPublicInput<F: PrimeField>(Vec<F>);
+
+/// Concrete public input type
+pub type PublicInput = GenericPublicInput<CircuitField>;
+
```

Comment:
can we move up this, so that all concrete types are in the same place, maybe better yet just move concrete type alias to a separate file?